internal/telegram/telegram.go

```diff
@@ -0,0 +1,63 @@
+package telegram
+
+import (
+	"fmt"
+	"io"
+	"log/slog"
+	"net/http"
+	"net/url"
+)
+
+var logger = slog.With("service", "telegram")
+
+func Start() {
+	logger.Info("starting telegram service")
+	loadEnvs()
+}
+
+func Stop() {
+	logger.Info("stopping telegram service")
+}
```

Comment:
I'm considering creating a `Service` interface to facilitate the creation of services bootstrap and teardown.

This is something that would require a lot of refactoring, so I'm planning to do this in future, when we have more services to manage